jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -157,18 +160,23 @@ async def _load_balance(self, request):
         try:
             async with aiohttp.ClientSession() as session:
 
-                if request.method == 'GET':
-                    request_kwargs = {}
-                    try:
-                        payload = await request.json()
-                        if payload:
-                            request_kwargs['json'] = payload
-                    except Exception:
-                        self.logger.debug('No JSON payload found in request')
```

Comment:
Looking at the original implementation, i have this idea:
it looks like the original logic was only to write a debug log which is not useful at all for production application. Can we just act as a pure proxy here for performance consideration? Something like:
```
async with session.request(request.method, data=request.iter_any(), **request_kwargs) as response:
    ....
```

@NarekA @JoanFM

Reply:
For some reason, if I try to pass the content in any other way besides the json field, I get an error here. I've tried everything at this point, if you can get this to work, I am interested.

Reply:
what error do you see?